### ISSUE TITLE
Weekly digest of not responded issues

### ISSUE THREAD

**Issue by karelz** [karelz/GitHubIssues#4]
Send weekly report with all issues in area which has not been answered by MS employee (or area owner - should it be a config?) for last ~3 days.

Bonus: also for closed issues

**Comment by karelz** [comment]
Depends on #3 to allow download of more data from GitHub by the tool.

**Comment by karelz** [comment]
Note: RedHat folks are treated as area owners as well (Linux alert vs. Networking alert), we might want to exclude them and separate non-area alerts (2.0, Linux)

**Comment by karelz** [comment]
@ianhays did you enable it also for Closed issues? If not, let's create new issue (P3).

**Comment by ianhays** [comment]
I purposefully left it out for closed issues. To add unresponded comment support for closed issues we would need to query the Github API for each of those issues (roughly 5000) which would hit [the API limit even when authenticated](https://developer.github.com/v3/#rate-limiting).

**Comment by karelz** [comment]
@ianhays could we first compare check last modified time to filter which issues to look at?
Would OAuth help us?

**Comment by ianhays** [comment]
OAuth would not make a difference, the limit is the same for any authentication type: 5000 per hour.

We could get comments for closed issues that were closed within the past X days, but I personally don't foresee that having much value.

The far more interesting data would be to only get comments for closed issues where the last comment occurred within the past X days, but that we cannot do without first getting all of the comments which hits the API limit.
